core/src/model/api/platform_objects.py

```diff
@@ -4,7 +4,7 @@
 
 
 @unique
-class Platform(str, Enum):
+class Platform(str, Enum):  # noqa: WPS600
```

Comment:
I'd prefer to disable rules which we are not going to fix in this PR, putting noqa everywhere is not a good practice